ci-operator/step-registry/openshift/e2e/aws/windows-node/openshift-e2e-aws-windows-node-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-e2e-aws-windows-node
+  steps:
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      OO_PACKAGE: windows-machine-config-operator
```

Comment:
where is this image from?
how does this command work for different openshift releases?

Reply:
> where is this image from?

See [optional-operators-subscribe](https://github.com/openshift/release/blob/2e0961ddf35d543ae5f37dd52034cf5d06b1be4b/ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-ref.yaml#L2) step

> how does this command work for different openshift releases?

Again the answer is in the [optional-operators-subscribe](https://github.com/openshift/release/blob/2e0961ddf35d543ae5f37dd52034cf5d06b1be4b/ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-ref.yaml#L2) step, where a day-2 operator is installed from a provided index image.